### DISCUSSION TITLE
Open all projects in a specific Tag in separate windows

### DISCUSSION THREAD

**Discussion by is-sam** [alefragnani/vscode-project-manager#666] [Ideas]
Whenever I need to switch project to open the frontend and backend projects, I have to do it manually. It would be great if it was possible to just right click a Tag and select "Open Group" or something so that all the projects within that tag opens in separate windows.

I would like to know if this feature is planned in the near future, if not I could fork the project and try to implement it. Let me know !

**Comment by alefragnani** [comment]
Hi @is-sam ,

No, there is no such feature. Right now, **tags** are just an organization/filtering feature. 

This has never been requested before, but yes, It's an interesting idea, and I guess it could be added in the future.

Thanks for your suggestion

**Reply by alefragnani** [reply]
Created https://github.com/alefragnani/vscode-project-manager/issues/667 to track this

**Comment by Turiok** [comment]
Hi everyone,

Is it possible to have the same request but with a button to open multiple projects in the same window with a multi-root vscode workspaces?

To explain my context :
I work with a lot of Git repositories and only in remote SSH.
I can't use this extension to switch project in vscode window or new window. Because the SSH connection is pretty slow.
I, mainly, work with Python and Poetry with virtual env for each repositories. But I can't create virtual env in the same place than the code because my storage is too limited. The virtual envs are in other storage. Yes, my company puts obstacles in my way.
And I have to specify virtual env for each repositories. For now, vscode accept virtual env only for root folder workspace.
I don't work with lots of vscode windows, otherwise I'd have 5 or 10 open all the time.

My ideal method of working would be : 
This plugin adding new Git repositories in the menu.
I can add tag for all repositories and open tag in a new window or the current with a multi root workspace for each repositories.
I think the possibility to click several project directly and open it in a new window to create a vscode workspace file would be great too.
